### PR TITLE
Adds PHP client examples

### PR DESCRIPTION
**Pull request by ecomba**
Adds examples for:

- list accounts
- auth
- bad auth
- check domains
- create contact
- create domain
- list domains
- get extended tld attributes
- list zone records

### CODE REVIEW

File: php/README.markdown [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# DNSimple API PHP Examples
+
+The scripts in this directory demonstrate how to use the DNSimple PHP API wrapper to connect to the DNSimple API.
+
+## Running
+
+First you will need to ensure PHP is installed and then run `php -v`.
+
+Once you install the dependencies, create a file called `token.php` and put the following into it:
```
Comment (ggalmazor) [inline php/README.markdown:9]: We should explain how to install the dependencies. I suggest we add a basic composer.json file at `/php` with `dnsimple/dnsimple` as the only dependency and then expand these instructions with something that explains how to get `composer.phar` and run `php composer.phar install`

File: php/README.markdown [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+```php
+<?php
+
+$token = "your_token";
+```
+
+Where `your_token` is an account token you generated on the DNSimple site from an account screen. Most of the example
```
Comment (ggalmazor) [inline php/README.markdown:17]: We should mention that we need a sandbox token.

**Review by ggalmazor — CHANGES_REQUESTED** [review]
I've run all the examples and found that all the comments are super useful to understand how to work with the PHP client.

I think we need to tweak the `README` file a bit to make it easier to run the examples from scratch. See my other comments :)

File: php/README.markdown (hunk 1 above)
Comment (ecomba) [reply php/README.markdown:9]: That's a great idea! 👍

File: php/README.markdown (hunk 2 above)
Comment (ecomba) [reply php/README.markdown:17]: Makes sense. I was following the way other examples are worded, but this makes a lot of sense!

File: php/README.markdown (hunk 1 above)
Comment (ecomba) [reply php/README.markdown:9]: Quick question though:

Don't you think that adding the composer.json file would be enough? Maybe stating in the readme that they could use composer to install the dependencies... I don't necessarily want to go through the lengths of explaining how to install composer, etc...

**Review by ggalmazor — APPROVED** [review]
LGTM! Thanks, @ecomba :)

**Pull request merged**